### PR TITLE
Add Ridge Racer 3D

### PR DESCRIPTION
**Pull request by brothe2nd**
Adds Ridge Racer 3D.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Leo626** [comment]
I agree. I'd like to see at least one screenshot of gameplay, then it'll LGTM.

**Comment by brothe2nd** [comment]
race

**Comment by j-selby** [comment]
@brothe2nd Put that in your PR, not in the comments 😥

File: games/ridge-racer-3d/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Ridge Racer 3D"
+description = "Ridge Racer 3D is a racing video game for the Nintendo 3DS which is published and developed by Namco Bandai Games. First images of the game were leaked in early November 2010, and Namco confirmed that the video game was going to be released later on in 2011."
```
Comment (TheKoopaKingdom) [inline games/ridge-racer-3d/game.dat]: The second sentence here isn't really necessary, it doesn't say much about the game.

**Comment by TheKoopaKingdom** [comment]
Please fix your main menu screenshot, and add your gameplay screenshot to the PR or else this will be closed.

**Comment by TheKoopaKingdom** [comment]
Closing this due to inactivity.

**Pull request closed without merging**